roles/reset/tasks/main.yml

```diff
@@ -107,6 +107,10 @@
   tags:
     - iptables
 
+- name: Clear IPVS virtual server table
+  shell: "ipvsadm -C"
+  when: clear_ipvs_vstable|bool
```

Comment:
1) when statement as list
```
when:
    - statement
```
2) it should detect and clear ipvs table when kube_proxy_mode is set to ipvs, and not introduce new variable